packages/store-sync/src/createStoreSync.ts

```diff
@@ -49,13 +50,15 @@ type CreateStoreSyncOptions<TConfig extends StoreConfig = StoreConfig> = SyncOpt
 export async function createStoreSync<TConfig extends StoreConfig = StoreConfig>({
   storageAdapter,
   onProgress,
-  address,
   publicClient,
+  address,
+  tableIds = [],
   startBlock: initialStartBlock = 0n,
   maxBlockRange,
   initialState,
   indexerUrl,
 }: CreateStoreSyncOptions<TConfig>): Promise<SyncResult> {
+  const includedTableIds = new Set(tableIds.length ? [...internalTableIds, ...tableIds] : []);
```

Comment:
it looks like we actively pass `internalTableIds` here, but also always include it internally in the query adapter - doesn't really hurt bc it's a set, but feels like it could be confusing (e.g. someone removing `internalTableIds` here and then wondering why they still show up in the results). What are your thoughts on only putting it in one of the two places and making it an explicit assumption? (e.g. "`findAll` always returns internal tables", or "`findAll` returns only the tables passed in the query, it is recommended internal tables are always included in the query")

Reply:
I think making the indexer dumber (uses whatever you pass in for table IDs) makes sense, and only the store-sync includes internal table IDs.